robot/impl/resource_manager_modular_test.go

```diff
@@ -208,7 +208,7 @@ func TestModularResources(t *testing.T) {
 
 		// Add a modular service
 		r.Reconfigure(context.Background(), &config.Config{
-			Components: []resource.Config{cfg},
+			Services: []resource.Config{cfg},
```

Comment:
do you think we're sufficiently testing both services and components at the same time in this test pkg?